lighthouse-core/audits/network-requests.js

```diff
@@ -63,6 +64,13 @@ class NetworkRequests extends Audit {
           granularity: 1,
           text: 'Transfer Size',
         },
+        {
+          key: 'resourceSize',
+          itemType: 'bytes',
+          displayUnit: 'kb',
```

Comment:
isn't this bytes?

![image](https://user-images.githubusercontent.com/4071474/51404819-6bdfde80-1b09-11e9-8190-f6e3bd7ebaf0.png)

Reply:
yeah, it's interpreted by the `itemType` but is put into the LHR as the `displayUnit`

Reply:
> but is put into the LHR as the displayUnit

er, that's not right. It is if it's in an i18ned string, otherwise it's not until `details-renderer.js` that the `displayUnit` is used...which is pointless here because these aren't displayed in the html report